src/components/MdDatepicker/MdDatepicker.vue

```diff
@@ -28,25 +28,10 @@
   import MdOverlay from 'components/MdOverlay/MdOverlay'
   import MdDatepickerDialog from './MdDatepickerDialog'
   import MdDateIcon from 'core/icons/MdDateIcon'
+  import debounce from 'core/utils/MdDebounce'
```

Comment:
It should be renamed from `debounce` to `MdDebounce` because it can later conflict with other libraries.